src/clj/puppetlabs/services/jruby_pool_manager/impl/jruby_internal.clj

```diff
@@ -18,7 +18,8 @@
            (com.puppetlabs.jruby_utils.jruby InternalScriptingContainer
                                              ScriptingContainer)
            (org.jruby.runtime.profile.builtin ProfileOutput)
-           (java.io File)))
+           (java.io File)
+           (org.jruby.util KCode)))
```

Comment:
nit: if we're changing this can we put the imports in alphabetical order?

Reply:
Updated